__tests__/app.test.js

```diff
@@ -100,24 +97,53 @@ describe('GET', () => {
                 return request(app)
                 .get('/api/articles/9999')
                 .expect(400)
-                .then(({body}) => {
-                    expect(body.msg).toBe('Bad Request');
+                .then(({body: { msg }}) => {
+                    expect(msg).toBe('Bad Request');
                 });
             });
 
             test('400: /api/articles/not_an_id When passed an id that is not the correct format send a 400 bad request', () => {
                 return request(app)
                 .get('/api/articles/not_an_id')
                 .expect(400)
-                .then(({body}) => {
-                    expect(body.msg).toBe('Bad Request');
+                .then(({body: { msg }}) => {
+                    expect(msg).toBe('Bad Request');
                 });
             });
 
         });
 
     });
 
+    describe('GET: /api/articles', () => {
+
+        describe('GOOD PATH:', () => {
+
+            test(`200: /api/articles responds with an array of article objects: 
+            each having author, title, article_id, topic, created_at, votes, 
+            article_img_url, comment_count properties`, () => {
+                return request(app)
+                .get('/api/articles')
+                .expect(200)
+                .then(({body: { articles }}) => {
+                    expect(Array.isArray(articles)).toBe(true);
+                    articles.forEach((article) => {
```

Comment:
There should also not be a body property in this object. Consider how you'd test this